src/js/components/DataTable/DataTable.js

```diff
@@ -219,62 +223,35 @@ const DataTable = ({
     console.warn('DataTable cannot combine "size" and "resizeble".');
   }
 
+  const [page, showItem] = normalizeShow(showProp, 'dataTable');
+  const [setPage, currentItems, currentPage] = usePagination({
```

Comment:
I think a return value like `[items, page, setPage]` would be more consistent with the other hooks.

Reply:
Done, returns [items, paginationProps].